releasenotes/notes/add-missing-device_name-tags-07e712f31b5a774d.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - Add missing `device_name` tags on docker, containerd et network checks.
```

Comment:
Nit: the format of `.rst` files is not markdown, please see release notes entries (on top of my mind but I'm not a specialist: two quotes for "code escaping" and no ` -` to enumerate)
`et` → `and`